### DISCUSSION TITLE
Parameterisation of hygiene tests

### DISCUSSION THREAD

**Discussion by mereolog** [edmcouncil/ontology-publisher#44] [Code Improvements]
@rivettp has raised a reasonable, in my view, request that hygiene tests should be parameterised - see: https://github.com/edmcouncil/fibo/pull/1199#pullrequestreview-513211941.

For now just sending the 'edmcouncil' parameter should be sufficient.

**Comment by przemekgradzki** [comment]
_ontology-publisher_ should be generalized. So why should we include an "edmcouncil" parameter related to a particular ontology?
Currently the [hygiene tests](https://github.com/edmcouncil/fibo/tree/master/etc/testing/hygiene) are in the same repository as the [fibo ontology](https://github.com/edmcouncil/fibo/). Are there plans to generalize the hygiene tests and put them in this repository? Or in some separate repository?

**Comment by trypuz** [comment]
@przemekgradzki , @mereolog , please try to be as general as possible. Perhaps we should have a more general parameter for an ontology we are testing.
